examples/meta/CMakeLists.txt

```diff
@@ -23,8 +23,18 @@ LIST(APPEND GENERATOR_DEPENDENCIES ${CMAKE_SOURCE_DIR}/examples/meta/generator/t
 FILE(GLOB TARGET_LANGUAGES ${CMAKE_SOURCE_DIR}/examples/meta/generator/targets/*.json)
 LIST(APPEND GENERATOR_DEPENDENCIES ${TARGET_LANGUAGES})
 
-# parser output
-file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/parser_files)
+# parser generation
+set(GENERATED_PARSER_FILES)
+LIST(APPEND GENERATED_PARSER_FILES ${CMAKE_CURRENT_BINARY_DIR}/parser_files/parsetab.py)
+LIST(APPEND GENERATED_PARSER_FILES ${CMAKE_CURRENT_BINARY_DIR}/parser_files/lextab.py)
+add_custom_command(
+    OUTPUT ${GENERATED_PARSER_FILES}
+    DEPENDS ${GENERATOR_DEPENDENCIES}
```

Comment:
`parse.py` does not depend on any generator dependencies except `${CMAKE_CURRENT_SOURCE_DIR}/generator/parse.py`.

Reply:
You're right. Didn't read what was in the generator dependencies list properly. Thanks for feedback!